src/github/api/mod.rs

```diff
@@ -17,6 +18,9 @@ use crate::permissions::TeamApiClient;
 pub mod client;
 pub(crate) mod operations;
 
+// timeout in seconds
+const TIMEOUT: u64 = 10;
```

Comment:
```suggestion
const HTTP_TIMEOUT: Duration = Duration::from_secs(10);
```
And then we don't even need the comment :)